app/views/students/_student_form.html.erb

```diff
@@ -21,6 +21,11 @@
                   remote: true,
                   class: 'button inline-button' %>
     </span>
+    <%= f.label :hidden, t('students.update_status') %>
+    <span id='set-active'>
```

Comment:
I'm not sure why you included a `span` here, the layout should be okay with just the `select` tag (I think)?

Reply:
We have styling in place that will fill the select with more space without the span tag. Removing the tag in this case would look like:
![Capture](https://user-images.githubusercontent.com/67441706/183230196-766cc92c-4b4c-41c4-b252-3e866d38abdc.PNG)
I feel it's easier to just have the tag here.